src/main/java/frc/robot/util/JoystickModification.java

```diff
@@ -0,0 +1,73 @@
+package frc.robot.util;
+
+import java.util.Map;
+
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.wpilibj.shuffleboard.BuiltInWidgets;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+
+public class JoystickModification {
```

Comment:
This is a nice encapsulation of related code from RobotContainer. Helps to keep that other class focused on robot integration / composition and this new class for joystick considerations